tests/cmaps/test_get_cmap.py

```diff
@@ -24,19 +30,18 @@ def test_get_cmap():
 
 
 def test_get_cmap_filesystem(monkeypatch):
-    import pkg_resources
     import importlib
 
     import terracotta.cmaps.get_cmaps
 
     def throw_error(*args, **kwargs):
-        raise pkg_resources.DistributionNotFound("monkeypatched")
+        raise ModuleNotFoundError
```

Comment:
```suggestion
        raise ModuleNotFoundError("monkeypatched")
```

Reply:
Huh! Indeed - I missed that - I modified the test a bit so it would have actually failed on me.